test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -311,6 +323,36 @@ def test_dependency_check_neuron(neuron, ec2_connection):
     _run_dependency_check_test(neuron, ec2_connection, "neuron")
 
 
+@pytest.mark.model("N/A")
+def test_dataclasses_check(image):
+    """
+    Ensure there is no dataclasses pip package is installed for python 3.7 and above version.
+    :param image: ECR image URI
+    """
+    ctx = Context()
+    pip_package = "dataclasses"
+
+    container_name = get_container_name("dataclasses-check", image)
+
+    python_version = re.findall(r'py(\d\.?\d?)', image)[0]
```

Comment:
can you use test_utils.get_python_version_from_image_uri here?

Reply:
ack